services/pane.js

```diff
@@ -148,6 +151,28 @@ function openPublish() {
   });
 }
 
+/**
+ * open new page type dialog pane
+ * @returns {Promise}
+ */
+function openNewPage() {
+  var header = 'Select New Page Type',
```

Comment:
Yeah! Too wordy. And "type" is robotic. Since there are options, how about "Select New Page" since we are prompting the user to make a decision?